### PR TITLE
Add LVM test

### PR DESCRIPTION
**Pull request by paolostivanin**
Add lvm cli test (I took over PR#7409 which can now be closed)

- Related ticket: https://progress.opensuse.org/issues/51161
- Verification runs:
  - oS TW: http://d250.qam.suse.de/tests/2054 :heavy_check_mark: 
  - oS Leap: http://d250.qam.suse.de/tests/2056 :heavy_check_mark: 
  - SLE 15.0: http://d250.qam.suse.de/tests/2052 :heavy_check_mark: 
  - SLE 12.4: http://d250.qam.suse.de/tests/2051 :heavy_check_mark: 
  - SLE 12.2: http://d250.qam.suse.de/tests/2050 :heavy_check_mark:

### CODE REVIEW

File: tests/console/lvm.pm [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    # move data from the original extend to the new one
+    validate_script_output("pvcreate ${disk}3",      sub { m/successfully created/ },  180);
+    validate_script_output("vgextend test ${disk}3", sub { m/successfully extended/ }, 180);
+    assert_script_run "pvmove ${disk}1 ${disk}3";
+
+    # after moving data, remove the old extend with no data
+    validate_script_output("vgreduce test ${disk}1", sub { m/Removed/ }, 180);
+
+    # check the data just to be sure
+    assert_script_run 'cat /mnt/test_lvm/test | grep test';
```
Comment (asmorodskyi) [inline tests/console/lvm.pm]: why not to use validate_script_output here too ?

File: tests/console/lvm.pm [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    assert_script_run 'mount /dev/test/one /mnt/test_lvm';
+    assert_script_run 'cat /mnt/test_lvm/test|grep test';
+
+    # extend the filesystem
+    validate_script_output("xfs_growfs /mnt/test_lvm", sub { m/data blocks changed/ }, 180);
+    validate_script_output("df -h /mnt/test_lvm", sub { m/test/ }, 180);
+    assert_script_run 'cat /mnt/test_lvm/test | grep test';
+
+    # move data from the original extend to the new one
+    validate_script_output("pvcreate ${disk}3",      sub { m/successfully created/ },  180);
```
Comment (asmorodskyi) [inline tests/console/lvm.pm]: does it make sense to extract timeout to a variable ?

**Comment by paolostivanin** [comment]
run with latest change is green: http://d250.qam.suse.de/tests/2060 :heavy_check_mark:

**Comment by paolostivanin** [comment]
ping @asmorodskyi :)

**Pull request merged**